app/controllers/api/v21/models_controller.rb

```diff
@@ -3,12 +3,6 @@ module V21
     class ModelsController < V21::BaseController
       before_filter :find_resource, :only => %w{show update destroy}
 
-      api :GET, "/models/", N_("List all hardware models")
```

Comment:
removed for controllers that aren't used by fusor and only added `domains` and `hostgroups` that are used by fusor